include/functions.h

```diff
@@ -1364,14 +1364,14 @@ void View_InitDistortion(View* view);
 void View_ClearDistortion(View* view);
 void View_SetDistortion(View* view, Vec3f orientation, Vec3f scale, f32 speed);
 s32 View_StepDistortion(View* view, Mtx* projectionMtx);
-void func_800AAA50(View* view, s32 arg1);
-s32 func_800AAA9C(View* view);
-s32 func_800AB0A8(View* view);
-s32 func_800AB2C4(View* view);
-s32 func_800AB560(View* view);
-s32 func_800AB944(View* view);
-s32 func_800AB9EC(View* view, s32 arg1, Gfx** p);
-s32 func_800ABE74(f32 eyeX, f32 eyeY, f32 eyeZ);
+void View_Apply(View* view, s32 mask);
+s32 View_ApplyPerspective(View* view);
+s32 View_ApplyOrtho(View* view);
+s32 View_ApplyToOverlayOrtho(View* view);
+s32 View_ApplyToOverlayPerspective(View* view);
+s32 View_UpdateViewingMatrix(View* view);
+s32 View_ApplyTo(View* view, s32 arg1, Gfx** p);
+s32 View_SanityCheckEyePosition(f32 eyeX, f32 eyeY, f32 eyeZ);
```

Comment:
Purely a preference suggestion: `ErrorCheck`?

Reply:
I do quite like the slight extra information you get from the `SanityCheck` name. It's specifically checking for enormous values — the eye position isn't invalid, exactly, it's just *insane*. I think this tracks with the general use of "sanity check" to mean checking for values that may not crash or anything but are unlikely to be what you wanted.

Reply:
I see, I'm used to viewing numbers outside of [-32768, 32767] as more of an error than just a bogus value, since the geometry transformation code that runs on the RSP processor actually cannot handle values outside of this range properly, they would get truncated when loading into the 8x16b vector registers. This still doesn't crash though, so it's probably not any worse than what you already get with such large eye values, so it can go either way really.

Reply:
that sounds like enough of a reason to call it an error to me either way though, if the rsp cant handle those numbers

Reply:
Didn't know that about the RSP! Will change.

Reply:
1719252db